eth/backend.go

```diff
@@ -192,6 +192,7 @@ type Ethereum struct {
 	blockWriter    *blockio.BlockWriter
 	kvRPC          *remotedbserver.KvServer
 	logger         log.Logger
+	closeCh        chan struct{} // Channel to signal the background processes to exit
```

Comment:
Since you have already this bor struct we don't need it here

Reply:
Removed